demonstrations/tutorial_how_to_use_arithmetic_operators.py

```diff
@@ -0,0 +1,357 @@
+r"""
+How-to use Quantum Arithmetic Operators
+=======================================
+
+Classical computers handle arithmetic operations like addition, subtraction, multiplication, and exponentiation with ease. 
+For instance, you can multiply large numbers on your phone in milliseconds!
+
+Quantum computers can handle these operations too, but their true value lies beyond basic calculations. Quantum arithmetic plays a crucial role in more advanced quantum algorithms, 
+serving as fundamental building blocks in their design and execution. For example:
```

Comment:
Can you add half a sentence here explaining what this crucial role is? As in, why the reader should care about this Demo?

Reply:
```suggestion
serving as fundamental building blocks in their design and execution, and knowing your way around quantum arithmetic with PennyLane could be just the thing to take your algorithms from simple to complex. For example:
```
Is this what you would like to say? I just think it needs a bit of a 'hook', you know?

Reply:
I see your point now. While I prefer to keep the original sentence before the examples as it is, we can definitely add a 'hook' after the examples. I suggest the following change:

With PennyLane, you will see how easy it is to build quantum arithmetic operations as subroutines for your quantum algorithms! -> With PennyLane, you’ll see how easy it is to build quantum arithmetic operations as subroutines for your algorithms. Knowing your way around these operators could be just the thing to streamline your algorithm design!

Reply:
Yeah, why not — if you're happy with this, that's great.